cvpy/annotation/base/Credentials.py

```diff
@@ -0,0 +1,86 @@
+import os
+import stat
+from pathlib import PosixPath
+
+
+class Credentials(object):
+
+    def __init__(self, user: str = None, password: str = None, token: str = None, auth_file: str = None) -> None:
```

Comment:
Should it be "user" or "username"? Or something else like "login"? I haven't seen just "user" used in authentication.

Reply:
I changed it to username which is consistent with swat CAS connection parameter.